src/easystar.js

```diff
@@ -159,6 +159,18 @@ EasyStar.js = function() {
         directionalConditions[y][x] = allowedDirections;
     };
 
+    /**
+    * Get the directional conditions previously set on a tile
+    *
+    * @param {Number} x The x value of the point.
+    * @param {Number} y The y value of the point.
+    * @return {Number} A list of all the allowed directions that can access
+    * the tile.
+    **/
+    this.getDirectionalCondition = function(x, y) {
```

Comment:
Did you intend to include this as part of this PR?

Reply:
No that was a mistake, my bad!

Reply:
But this is a feature missing from the library by the way

Reply:
Ah ok. We could consider adding it in another PR.